zeus/optimizer/batch_size/server/batch_size_state/repository.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import annotations
```

Comment:
There should be a newline separating imports from `__future__` and imports from built-in modules.